pkg/plugins/golang/deploy-image/v1alpha1/api.go

```diff
@@ -69,6 +69,9 @@ type createAPISubcommand struct {
 
```

Comment:
missing update the e2e tests, see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/deployimage/plugin_cluster_test.go#L108

We need to pass the runAsUser now there